tests.props

```diff
@@ -0,0 +1,11 @@
+ch.qos.logback:* = 1.2.3
+io.dropwizard:* = 1.3.9
+io.dropwizard.metrics:* = 4.0.5
+io.github.openfeign:feign-core = 9.5.0
+javax.validation:validation-api = 2.0.1.Final
```

Comment:
This is 1.1.0.Final according to the Dropwizard table - is this intentional?

Reply:
Hmm, I'm not sure why I originally ended up bumping this.  I'll try dropping it back down.

Reply:
Meh, guess it wasn't necessary 🤷‍♂